kayenta-core/src/test/groovy/com/netflix/kayenta/util/RetrySpec.groovy

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.kayenta.util
+
+import spock.lang.Specification
+import spock.lang.Unroll;
+
+class RetrySpec extends Specification {
```

Comment:
This file ended up with 4-space indentation instead of 2-space.

Reply:
Fixed.